client-rpc/src/rpc/multisig_rpc.rs

```diff
@@ -5,13 +5,23 @@ use jsonrpc_derive::rpc;
 use secstr::SecUtf8;
 
 use chain_core::common::{H256, HASH_SIZE_256};
+use chain_core::init::address::CroAddress;
 use client_common::{Error, ErrorKind, PublicKey, Result as CommonResult};
 use client_core::{MultiSigWalletClient, WalletClient};
 
-use crate::server::{to_rpc_error, WalletRequest};
+use crate::server::{rpc_error_from_string, to_rpc_error, WalletRequest};
 
 #[rpc]
 pub trait MultiSigRpc: Send + Sync {
+    #[rpc(name = "multiSig_createAddress")]
+    fn create_address(
+        &self,
+        request: WalletRequest,
+        public_keys: Vec<String>,
+        self_public_key: String,
+        required_signatures: usize,
```

Comment:
usize is ambious type,
how about u32

Reply:
Yes, I think a more concrete integer size maybe appropriate. Any reason to use usize before @devashishdxt

Reply:
There is no specific reason. The only reason is that Rust's `Vec` length is measured in `usize`. I don't think there is any reason to change it to anything else since we already have a limit on number of leaf nodes a merkle tree can have.